CONTRIBUTING.md

```diff
@@ -1,14 +1,20 @@
 # Contributing to Rust IPFS
 
-Welcome, and thank you for your interest in contributing to Rust IPFS. Issues and pull requests are encouraged. As we work through the efforts described in the approved IPFS Dev Grant, we're looking for help in the following
+Welcome, and thank you for your interest in contributing to Rust IPFS. Issues and pull requests are encouraged. You can make a difference by tackling any of the following items:
 
-* Implementing endpoints and features not covered in the dev grant (See the [README](./README.md#roadmap))
-* Tests and CI for existing functionality
-* Examples
+* Implementing the remaining features and corresponding HTTP endpoints not yet covered. See https://areweipfsyet.rs for info on which those are.
```

Comment:
I will merge this list with the one on the README. From there, I can either:

1. Move the entire list to CONTRIBUTING.md
2. Move the entire list to README.md
3. Duplicate the list and have it in both places.

What do you think?

Reply:
I'd say CONTRIBUTING.md, but I don't have strong feelings one way or the other

Reply:
Ok, moved it to CONTRIBUTING.md :+1: